UnitsNet/UnitDefinitions/Duration.json

```diff
@@ -162,6 +162,18 @@
           "Abbreviations": [ "нс" ]
         }
       ]
+    },
+    {
+      "SingularName": "Cycle",
```

Comment:
OK, so this has all spawned from the requirement to convert between 'seconds' and 'cycles' in our equations
=>where, given a system with a frequency of 50Hz 1 cycle = 1/frequency = 1/50 = 0.02 seconds.

So if we have the variable:

A = 20 cycles

I need to convert this variables units and end up with:

A = 20 * 0.02 = 0.4 seconds

So our equations literally have the unit 'cycle' and 'cycles' in them which I need to fit into the Units.Net framework somehow...

To put it in Frequency implies you can scale between a 'cycle' and 'hertz' but you cant do this without knowing how many seconds one cycle takes

Your suggestion of the Period extension method on the Frequency class makes sense - this will provide us the time taken for one cycle, thus from that we can convert between Time and Cycles by:

`TimeSpan seconds = Frequency.FromHertz(60).Period * NumberOfCycles`

The way I've been running my conversions is like this e.g. between a source frequency to a target frequency:

```
	Frequency sourceFrequency;
            if (Frequency.TryParse(sourceQuantity, out sourceFrequency))
            {
                FrequencyUnit targetFrequencyUnit;
                if (FrequencyWrapper.TryParseUnit(targetUom, out targetFrequencyUnit))
                {
                    switch (targetFrequencyUnit)
                    {
                        case FrequencyUnit.Undefined:
                            break;
                        case FrequencyUnit.Hertz:
                            result = sourceFrequency.Hertz;
                            return true;
			// and so on...
```

So which Unit definition should hold the 'cycle' unit?

And how do we structure the conversion between Cycles and Time?

I like your suggestion of getting the Frequency per country - but where in the Units.Net architecture would this reside? Would you want to see the Frequencies per country setup in a .config file somewhere - or a constants file?